src/modules/weather/api/weather-routes.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+import { getWeather } from '../utils/getWeather';
+
+module.exports = function (app: core.Express) {
+    /**
+     * @openapi
+     * '/weather/':
+     *   get:
+     *     tags:
+     *     - Weather
```

Comment:
Can you please also add this tag to the utils/swagger.ts file please.

Reply:
Adjusted.